nodelet/pkg/pf9kube/pf9/pf9-kube/utils.sh

```diff
@@ -453,6 +453,9 @@ staticPodPath: "${STATIC_POD_PATH}"
 tlsCertFile: "${TLS_CERT_FILE}"
 tlsPrivateKeyFile: "${TLS_PRIVATE_KEY_FILE}"
 tlsCipherSuites: ${TLS_CIPHER_SUITES}
+imageMinimumGCAge: 24h
+imageGCHighThresholdPercent: 95
```

Comment:
So what happens if this threshold is reached but no image is more than 24 hours old?

Reply:
Nothing happens. The garbage collector does not kick in and we must rely on the user to clean up images that aren't needed.

Reply:
Or find other ways to free up disk space.